kubejs/server_scripts/mods/modern_industrialization/chemical_reactor.js

```diff
@@ -30,6 +30,56 @@ ServerEvents.recipes(e => {
         e.custom(newRecipe).id(id);
     }
 
+    // -- BLAZE ROD -- //
+    chemicalReactor(
+        st('blaze_rod'),
+        16,
+        200,
+        [ 
+            { amount: 1, item: mc('stick') },
```

Comment:
change to `mi('steel_rod')`

why? funny -> :trollface: (i think it's more interesting than just a regular stick)